src/js/common/schemaform/widgets/SSNWidget.jsx

```diff
@@ -11,7 +11,11 @@ export default class SSNWidget extends React.Component {
     this.state = { val: props.value };
   }
   handleChange = (val) => {
-    const strippedSSN = val.replace(/[\- ]/g, '');
+    let strippedSSN;
+    if (val) {
+      strippedSSN = val.replace(/[\- ]/g, '');
+    }
```

Comment:
I just mean we can initialize a component with `val=''`, but once we modify the value, it can never get back to `''`. Might lead to some weird cases is all, and I didn't know if there was a better way to ensure `''` is always translated to `undefined`.